tslint.json

```diff
@@ -8,6 +8,7 @@
         "no-empty-interface": false,
         "no-unnecessary-callback-wrapper": false,
         "no-unnecessary-generics": false,
+        "no-unsafe-any": true,
```

Comment:
Manual testing didn't pick this bug up because this rule is disabled by default.